src/pages/iou/MoneyRequestCreatedPage.js

```diff
@@ -0,0 +1,120 @@
+import React, {useEffect} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import lodashGet from 'lodash/get';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithBackButton from '../../components/HeaderWithBackButton';
+import Form from '../../components/Form';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import * as IOU from '../../libs/actions/IOU';
+import optionPropTypes from '../../components/optionPropTypes';
+import NewDatePicker from '../../components/NewDatePicker';
+import useLocalize from '../../hooks/useLocalize';
+
+const propTypes = {
+    /** Onyx Props */
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
+    iou: PropTypes.shape({
+        id: PropTypes.string,
+        amount: PropTypes.number,
+        comment: PropTypes.string,
+        created: PropTypes.string,
+        participants: PropTypes.arrayOf(optionPropTypes),
+        receiptPath: PropTypes.string,
+    }),
+
+    /** Route from navigation */
+    route: PropTypes.shape({
+        /** Params from the route */
+        params: PropTypes.shape({
+            /** Which field we are editing */
+            field: PropTypes.string,
+
+            /** reportID for the "transaction thread" */
+            threadReportID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    iou: {
+        id: '',
+        amount: 0,
+        comment: '',
+        created: '',
+        participants: [],
+        receiptPath: '',
+    },
+};
+
+function MoneyRequestCreatedPage({iou, route}) {
```

Comment:
NAB: I know that `created` here refers to the field `created`. But I think that the file name suggests that it's a page to create a money request.

What do you think of `MoneyRequestDatePage` ?

Reply:
I was debating which one to use as I want to avoid the confusion of `created`/ `date` but I will probs change it to Date as you suggest, thanks